### PR TITLE
feat(discover2) Improve Contexts tab

### PR DESCRIPTION
**Pull request by markstory**
Show the user, trace and os context in more detail making useful information easier to get at.

Screen Shot 2019-08-30 at 2 04 14 PM

Fixes SEN-999

### CODE REVIEW

File: src/sentry/static/sentry/app/components/events/contexts.jsx [hunk 1]
```diff
@@ -51,17 +51,21 @@ class ContextChunk extends React.Component {
     this.syncPlugin();
   }
 
-  componentDidUpdate(prevProps, prevState) {
+  componentDidUpdate(prevProps) {
     if (
-      prevProps.group.id !== this.props.group.id ||
-      prevProps.type !== this.props.type
+      prevProps.type !== this.props.type ||
+      (prevProps.group && this.props.group && prevProps.group.id !== this.props.group.id)
```
Comment (scefali) [inline src/sentry/static/sentry/app/components/events/contexts.jsx]: @markstory so if `prevProps.group` is falsy, we don't ever want to call `syncPlugin`?

Comment (markstory) [reply src/sentry/static/sentry/app/components/events/contexts.jsx]: Good catch, we want to sync if the new state is different than before. I'll get that fixed.

Comment (scefali) [reply src/sentry/static/sentry/app/components/events/contexts.jsx]: @markstory In that case, you could use `_.get` here on `group.id` to simplify the code.

**Review by billyvg — APPROVED** [review]
LGTM

**Pull request merged**